docker/docker-compose.prod.yml

```diff
@@ -7,7 +7,8 @@ services:
     build:
       context: ../server
       dockerfile: Dockerfile
-    command: ["./start-server.sh"]
+    command: [ "start.sh", "immich" ]
```

Comment:
I'm not opposed to making this change, though it is unrelated and is incomplete as the old "start-server.sh" file is hanging around still.

Reply:
I was a bit reluctant to remove these files as it might break existing setups that have not yet been adapted to the changes introduced in https://github.com/immich-app/immich/pull/2689

Reply:
To be fair, this isn't the docker compose file included in the release, but to build production like containers locally. There is no issue with changing this imo.

Reply:
This change actually makes it consistent with the docker-compose.yml file.